client/src/pages/submitSecret/form.js

```diff
@@ -125,12 +163,14 @@ const Form = ({formResponse}) => {
           <input id="files" type="file" onChange={handleFileChange} className="form-control"/>
           <div id="files-list" className="row row-cols-4 g-1 mt-1"></div>
           <div className="col">
-            <div className="row">
-              <label for="passphrase" className="col-sm-2 col-form-label">Secret key</label>
-              <div className="col-sm-10">
+              <label id="div-input-pass-pub-label" for="passphrase" className="col-sm-2 col-form-label">Secret key</label>
+              <div id="div-input-pass-pub" className="col-sm-10">
                 <input type="password" id="passphrase" name="passphrase" onChange={handleInputChange} className="form-control"/>
               </div>
-            </div>
+              <div className='row'>
+                <button id="div-input-pass-pub-genkey" type="button" onClick={handleGenerateKeyPair} className="btn btn-primary d-block w-5" style={{visibility: 'hidden'}}>Generate Key Pair</button>
+                <a id="priv-key-copy-button" href='#' onClick={copyPrivateKey} style={{visibility: 'hidden'}}>Copy private key</a>
```

Comment:
Style. Inconsistent use of single and double quotes. Use single quotes

Reply:
Fixed in 0e48148